### PR TITLE
[docs] Configuration and note on IPC

### PR DESCRIPTION
**Pull request by ddhodge**
Add clarification that IPC and shared memory do not need to be configured, unlike regular postgres
DOC-116

@netlify /preview/deploy/manual-deployment/system-config/

### CODE REVIEW

File: docs/content/preview/deploy/manual-deployment/system-config.md [hunk 1]
```diff
@@ -11,7 +11,15 @@ menu:
 type: docs
 ---
 
-Do the following configuration steps on each of the nodes in the cluster.
+Perform the following configuration on each node in the cluster:
+
+- ntp and chrony
```
Comment (ddorian) [inline docs/content/preview/deploy/manual-deployment/system-config.md:16]:
```suggestion
- ntp or chrony
```
> chrony is an implementation of the Network Time Protocol. It is an alternative to ntpd, a reference implementation of NTP.

File: docs/content/preview/deploy/manual-deployment/system-config.md [hunk 2]
```diff
@@ -11,7 +11,15 @@ menu:
...
+- ulimits
+- transparent hugepages
+
+Keep in mind that, although YugabyteDB is PostgreSQL compatible and runs a postgres process, it is not a PostgreSQL distribution. The PostgreSQL it runs doesn't need the same OS and system resources that open source PostgreSQL requires. For this reason, the kernel configuration requirements are different.
+
+In particular, the main YugabyteDB process, the YB-TServer, is multi-threaded. As a result, you don't need to modify settings for shared memory and inter-process communication (IPC), because there is no inter-process communication or shared memory in a multi-threaded process model (all memory is shared by the same process).
```
Comment (ddorian) [inline docs/content/preview/deploy/manual-deployment/system-config.md:22]: This is true for yb-tserver. But each YSQL connection is still a separate postgresql process. 

The yb-tserver <---> postgres-process-per-ysql-connection is done through RPC or IPC (don't know if we've upgraded to IPC+shared_memory yet). You should ask someone from YSQL-team.

File: docs/content/preview/deploy/manual-deployment/system-config.md [hunk 3]
```diff
@@ -79,16 +86,15 @@ $ ulimit -n <value>
 -u (processes/threads): 64000
 ```
 
-{{< note title="Note" >}}
+{{< note title="Restart servers" >}}
 
-- After changing a ulimit setting, the YB-Master and YB-TServer servers must be restarted in order for the new settings to take effect. Check the `/proc/<process pid>` file to see the current settings.
-- Changes made using ulimit may revert following a system restart depending on the system configuration.
+After changing a ulimit setting, the YB-Master and YB-TServer servers must be restarted in order for the new settings to take effect. Check the `/proc/<process pid>` file to see the current settings.
```
Comment (ddorian) [inline docs/content/preview/deploy/manual-deployment/system-config.md:91]: We can mention here that ulimits are printed in yb-tserver.INFO file when it's started. 

That is the place to look to know for 100% certain that the limits are applied.

**Review by ddorian — CHANGES_REQUESTED** [review]
See some comments

File: docs/content/preview/deploy/manual-deployment/system-config.md (hunk 2 above)
Comment (ddorian) [reply docs/content/preview/deploy/manual-deployment/system-config.md:22]: Agree.

**Review by ddorian — APPROVED** [review]

File: docs/content/preview/deploy/manual-deployment/system-config.md [hunk 4]
```diff
@@ -173,38 +174,40 @@ You can check with the following command:
...
 
+You should consult your operating system documentation to determine the best way to modify a kernel command line argument for your operating system.
 
-Append "transparent_hugepage=always" to `GRUB_CMDLINE_LINUX` in `/etc/default/grub`
+On RHEL or CentOS 7 or 8, using grub2, the following steps are one solution:
 
-```sh
-GRUB_CMDLINE_LINUX="rd.lvm.lv=rhel/root rd.lvm.lv=rhel/swap ... transparent_hugepage=always"
-```
+1. Append "transparent_hugepage=always" to `GRUB_CMDLINE_LINUX` in `/etc/default/grub`
```
Comment (aishwarya24) [inline docs/content/preview/deploy/manual-deployment/system-config.md:191]:
```suggestion
1. Append "transparent_hugepage=always" to `GRUB_CMDLINE_LINUX` in `/etc/default/grub`.
```

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Pull request merged**